documentation/release-notes/topics/version_4.rst

```diff
@@ -2,12 +2,71 @@
+
+.. important::
+   1. Support for the *Windows* version of the Data Collector will end on *January 31, 2025*. Customers are encouraged to switch to the Linux version before then. Detailed :ref:`installation instructions <Setting-up-data-collectors-doc>` are in the product documentation.
+   2. `CentOS Linux 7 End of Life, is June 30, 2024 <https://blog.centos.org/2023/04/end-dates-are-coming-for-centos-stream-8-and-centos-linux-7/>`_. With that, installing the Data Collector will not be supported on this version of CentOS.
+
+=======================================
+Enhanced Licensed Feature: Connectivity
+=======================================
+- Connections and Circuits have been enhanced to allow users to attach documents, images, and links.
+
+
+========================================
+Enhanced Feature: Alarm Event Management
+========================================
+- A new *Assets -> Events* page was added to the application to allow for consolidated events management. The events displayed will be a global view of all events on assets the user can access. 
+
+- Users can export, filter, and sort the events list by various criteria.
+
+- Events can be acknowledged or closed individually and in bulk.
+
+=================================================
+Enhanced Feature: Notifications -> Alarm Policies
+=================================================
+- The notification template has been updated to aggregate multiple events in one email. This enhancement will reduce email noise in the case of event spikes.
+
+- Administrators can now select All or multiple asset types from the same Alarm Policy. Previously, users were allowed to choose a single asset type for policy.
+
+- Administrators can now select a notification channel for an Alarm Policy; more information on Notification Channels is provided below.
+
+========================================================
+Enhanced Feature: Notifications -> Notification Channels
+========================================================
+- This feature allows Administrators to create an Alarm Policy to channel notifications to an external communication or alerting system such as Microsoft Teams.
+
+- This release adds the integration with Microsoft Teams. Administrators can configure a link to a specific Microsoft Teams Channel. Administrators can add multiple Teams channels and target them with different Alarm Policies.
+
+================================================
+Enhanced Feature: Linux & Windows Data Collector
+================================================
+- VMware protocol has been enhanced to add a monitoring pipeline for discovered sensors.
+
+======================================
+Other notable changes and improvements
+======================================
+- Launch Web Interface has been added as a primary action button on device asset types. 
```

Comment:
We should mention the new volume locale setting

Reply:
Fixed.